shared/chat/conversation/messages/attachment/image/index.js

```diff
@@ -82,6 +84,17 @@ class ImageAttachment extends React.PureComponent<Props, State> {
           {this.props.showPlayButton && (
             <Icon type="icon-play-64" style={iconCastPlatformStyles(styles.playButton)} />
           )}
+          {this.props.showFilmButton && (
+            <Icon type="icon-film-64" style={iconCastPlatformStyles(styles.playButton)} />
+          )}
+          {this.props.videoDuration &&
```

Comment:
this is dangerous in react native as android will crash if this is a string and it gets passed down into the render. Instead do
```
{!!this.props.videoDuration &&
```

eventually we'll get flow to catch this but until then we just have to be careful